2021-12-23 15:50:08,637 - ERROR - Row 5
Traceback (most recent call last):
  File "C:\Users\Matthew Curtis\Documents\Github-Pipl\Tests\Full Featured Example\Python CSV script Example - Full featured.py", line 215, in do_work
    if phonenumbers.is_possible_number(phonenumbers.parse(input_data[IDX_PHONE])):
  File "C:\Program Files\Python39\lib\site-packages\phonenumbers\phonenumberutil.py", line 2848, in parse
    raise NumberParseException(NumberParseException.NOT_A_NUMBER,
phonenumbers.phonenumberutil.NumberParseException: (1) The string supplied did not seem to be a phone number.
2021-12-23 15:50:08,746 - ERROR - Row 9 - Some fields are unsearchable: [Phone(number='+15412860146')]
2021-12-23 15:50:08,758 - WARNING - Row 9 could not get API response
2021-12-23 15:50:08,758 - WARNING - Invalid phone: Row 13, Row ID 400,+124567

2021-12-23 15:50:09,409 - ERROR - Row 13 - Status Code:400 - The query does not contain any valid name/username/user_id/phone/email/address to search by
2021-12-23 15:50:09,409 - WARNING - Row 13 - API Warning: We could not parse a valid username or user_id from the provided url 'google.com'
2021-12-23 15:50:09,409 - WARNING - Row 13 could not get API response
2021-12-23 15:50:09,665 - WARNING - Row 3 - API Warning: ZIP code `83854` was added based on street and city.

2021-12-23 15:50:09,992 - WARNING - Row 7 - API Warning: ZIP code `83854` was added based on street and city.

2021-12-23 15:50:10,785 - WARNING - Row 4 - API Warning: ZIP code `94133` was added based on street and city.

2021-12-23 15:50:10,900 - WARNING - Row 6 - API Warning: ZIP code `94133` was added based on street and city.

2021-12-23 15:50:11,167 - WARNING - Row 2 - API Warning: ZIP code `94133` was added based on street and city.

